_example/hello-world/Makefile

```diff
@@ -10,16 +10,16 @@ pack:
 
 dbuild:
 	@docker run --rm \
-		-u ${shell id -u}:${shell id -g} \
```

Comment:
Grrr you are right, in OSX `-f` works and in POSIX it `c`. https://developer.apple.com/legacy/library/documentation/Darwin/Reference/ManPages/man1/stat.1.html 

BTW, one thing that is weird for me is the outputs you have when you do `make dbuild`. I see a `go install` !!! Any idea? It hasn't to do an install!